module/actor/actor-sheet.js

```diff
@@ -603,8 +603,36 @@ export class VampireActorSheet extends ActorSheet {
     } else {
       dice2 = this.actor.data.data.abilities[item.data.data.dice2].value
     }
-
-    const dicePool = dice1 + dice2
+    
+        let potency = this.actor.data.data.blood.potency
```

Comment:
I would prefer adding a power_value field to BLOOD_POTENCY instead of having a switch since we already populate blood potency data using that const.